superset/databases/utils.py

```diff
@@ -112,7 +112,9 @@ def make_url_safe(raw_url: str) -> URL:
     :param raw_url:
     :return:
     """
+
+    url = str(raw_url).strip()
```

Comment:
if `url` might not be a string, can you fix the typing of the function?

Reply:
Yeah, that's tricky. I found that sometimes we're passing in `Url` because this method has already previously been run on a uri before it was passed in here, but it would technically be better if this function were passed in a string. But since the typing is just a hint, we still need to provide our own guards against incorrect types. Do you think adding `Url` to the typing is still helpful or maybe a comment here?

Reply:
Actually now that I look at the function parameter types for the original make_url, it can take a URL and then just returns the same URL. Good catch. I think I'll do the same here.

Reply:
Thanks for making the change. We definitely want the type hints to match the actual types passed in. mypy should ideally catch this when type checking too, i'm wondering why that wasn't the case